web-client/src/pages/my-reviews.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import '../App.css';
+import { makeStyles } from '@material-ui/core/styles';
+import MyReviewsOneReview from '../components/MyReviewsOneReview'
+import MyReviewsCategories from '../components/MyReviewsCategories'
+import Search from '../components/Search'
+import picture from '../images/empty-image.png'
+
+function MyReviews(props) {
+    const styles = makeStyles({
+        main:
+        {
+            width: '80%',
+            margin: 'auto',
+        },
+        left:
+        {
+            paddingTop: '5%',
+            width: '65%', 
+            float: 'left',
+        },
+        right:
+        {
+            paddingTop: '5%',
+            width: '35%',
+            float: 'right',
+        },
+        h3:
+        {
+            paddingTop: '5%',
+            width: '100%',
+            float: 'left',
+            textAlign: 'left',
+            paddingLeft: '5%',
+        }
+    });
+
+    const style = styles();
+
+    const reviews = [
+        {
+            title: 'Importance of Golden Ratio in Mathematics',
+            categories: 'Mathematics',
+            data: '30/11/2020',
+            authors: 'John Doe, Sam Smith, Ashley Blue',
+            text: 'Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts. Separated they live in Bookmarksgrove right at the coast of the Semantics, a large language ocean. A small river named Duden flows by their place. ',
+            path: picture,
+            author: 'John Doe',
+            modificationDate: '12/12/2020',
+        }
+    ];
```

Comment:
can you please add more reviews? so we can see how it works

Reply:
Ok, no problem.